component/slos.libsonnet

```diff
@@ -82,14 +82,36 @@ local prometheusRule(name) =
         },
       },
     ],
+    'vshn-postgresql-ha': [
+      newSLO('uptime', 'vshn-postgresql-ha', params.slos.vshn.postgres.uptime) {
+        description: 'Uptime SLO for High Available PostgreSQL by VSHN',
+        sli: {
+          events: {
+            // The  0*rate(...) makes sure that the query reports an error rate for all instances, even if that instance has never produced a single error
+            error_query: '(sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNPostgreSQL", instances="3"}[{{.window}}]) or 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}])) by (service, namespace, name, organization, sla) or vector(0)) > 0',
```

Comment:
And what's with clusters that have `instances=2`? They are still HA.

Reply:
Ah missed that, will update right  away

Reply:
Updated to` !="1"`